sites/profoodworld/server/templates/website-section/global-50.marko

```diff
@@ -0,0 +1,40 @@
+import GAM from "../../../config/gam";
+import GCSE from "../../../config/gcse";
+
+$ const { config } = out.global;
+
+$ const type = "specGuide";
+$ const title = "Global 250";
```

Comment:
The template is `global-50` by the title is `Global 250` - I'm assuming it should say `Global 50`?